scripts/reviews/update_auth_groups.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Update: Did this in a [migration](https://github.com/chrisseto/osf.io/pull/26/files/#diff-dce5737a3e0d5a3f2ef57b96bec4edf1R14)